### PR TITLE
Slave node pre-conf script

### PR DESCRIPTION
**Pull request by mmbelyaev**
a simple piece of commands

### CODE REVIEW

File: slave_node_pre-script.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+USER=jenkins
+sudo apt-get install docker docker-compose -y
+wget https://releases.hashicorp.com/vagrant/2.2.5/vagrant_2.2.5_x86_64.deb
+sudo install ./vagrant_2.2.5_x86_64.deb -y
+sudo apt-get install virtualbox -y
+sudo adduser --system --home /home/${USER} --shell /bin/bash --ingroup docker ${USER}
+sudo mkdir -p /home/${USER}/.ssh/
+sudo chown ${USER} /home/${USER}/.ssh
+sudo echo 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIE3Xoaz78I7AmvoJkUiydOzifTduCX8Secp0iWokJLBx mbelyaev@mitsuoka' >> /home/${USER}/.ssh/authorized_keys
```
Comment (marvin4ward) [inline slave_node_pre-script.sh]: `mbelyaev@mitsuoka` is meanless here.

**Comment by marvin4ward** [comment]
Please add chown as owner of authorize will be root

File: slave_node_pre-script.sh (hunk 1 above)
Comment (mmbelyaev) [reply slave_node_pre-script.sh]: fixed

**Comment by mmbelyaev** [comment]
fixed previous issues

File: slave_node_pre-script.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+USER=jenkins
+sudo apt-get install docker docker-compose -y
+wget https://releases.hashicorp.com/vagrant/2.2.5/vagrant_2.2.5_x86_64.deb
+sudo install ./vagrant_2.2.5_x86_64.deb -y
+sudo apt-get install virtualbox -y
+sudo adduser --system --home /home/${USER} --shell /bin/bash --ingroup docker ${USER}
+sudo mkdir -p /home/${USER}/.ssh/
+sudo chown ${USER} /home/${USER}/.ssh
+sudo echo 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIE3Xoaz78I7AmvoJkUiydOzifTduCX8Secp0iWokJLBx' >> /home/${USER}/.ssh/authorized_keys
+sudo chown ${USER} home/${USER}/.ssh/authorized_keys
```
Comment (marvin4ward) [inline slave_node_pre-script.sh]: `/` missed at `home/`

**Pull request merged**